docs/docs/migration_notes.md

```diff
@@ -25,6 +25,14 @@ For this reason you need to apply #[utility] macro to functions which were origi
     }
 ```
 
+With this change the `UnconstrainedContext` has been renamed as `UtilityContext`.
+This led us to rename the `unkonstrained` method on `TestEnvironment` as `utility` so you will need to update your tests using that:
```

Comment:
I think it's on the cspell too

Reply:
Surprisingly it's not there
<img width="311" alt="image" src="https://github.com/user-attachments/assets/c3ba1210-60f2-41b6-b860-f1df23a2f1fd" />